src/nvim/tui/input.c

```diff
@@ -233,9 +237,12 @@ static void tk_getkeys(TermInput *input, bool force)
     }
   }
 
-  if (result != TERMKEY_RES_AGAIN || input->paste_enabled) {
+  if (result != TERMKEY_RES_AGAIN /*|| input->paste_started */) {
```

Comment:
No, but this probably isn't needed anymore(I don't think it is possible to have `result == TERMKEY_RES_AGAIN && input->paste_enabled` because `read_cb` will only push data up to the next `\x1b`(esc). I'm not sure why this is there, probably an oversight or leftover from previous code.